### DISCUSSION TITLE
Is there an option to reverse the tilt direction?

### DISCUSSION THREAD

**Discussion by szymon-zadworny** [OpenTabletDriver/OpenTabletDriver#2292] [Q&A] [answered]
I have a Huion Q11K V2 tablet and the tilt direction is reversed when using Krita. I can set the proper tilt direction manually for each brush, but it'd be nice to be able to set it system-wide on a driver level.

Default tilt direction in Krita:
![Zrzut ekranu z 2022-06-04 12-46-42](https://user-images.githubusercontent.com/102227406/171995824-c0883716-2e28-44dd-af32-3f8f1a522388.png)

Fixed tilt direction:
![Zrzut ekranu z 2022-06-04 12-46-54](https://user-images.githubusercontent.com/102227406/171995875-1aab0f9b-0acd-4015-975e-a9a1032e348b.png)

**Comment by jamesbt365** [comment, marked as answer]
#1841?

**Reply by szymon-zadworny** [reply]
I don't know if this is it, but when I opened the debugger the Y tilt direction was indeed inverted (upwards tilt generated positive values instead of negative)
